source/features/update-pr-from-base-branch.tsx

```diff
@@ -42,27 +41,46 @@ async function handler(): Promise<void> {
 	});
 }
 
-async function addButton(position: Element): Promise<void> {
+async function addButton(mergeBar: Element): Promise<void> {
+	if (!select.exists(canMerge) || select.exists(canNativelyUpdate)) {
+		return;
+	}
+
 	const {base, head} = getBranches();
 	const prInfo = await getPrInfo(base.local, head.local);
-	if (!prInfo) {
+	if (!prInfo?.viewerCanEditFiles || prInfo.mergeable === 'CONFLICTING') {
 		return;
 	}
 
-	if (prInfo.viewerCanEditFiles && prInfo.mergeable !== 'CONFLICTING') {
-		position.append(' ', (
-			<span className="status-meta d-inline-block rgh-update-pr-from-base-branch">
-				You can <button type="button" className="btn-link">update the base branch</button>.
-			</span>
-		));
+	const mergeabilityRow = selectHas('.branch-action-item:has(.merging-body)')!;
+	if (mergeabilityRow) {
+		// The PR is not a draft
+		mergeabilityRow.prepend(
+
+			<div
+				className="branch-action-btn float-right js-immediate-updates js-needs-timeline-marker-header"
+			>
+				<button type="button" className="btn rgh-update-pr-from-base-branch">Update branch</button>
+			</div>,
+		);
+		return;
 	}
+
+	// The PR is still a draft
+	mergeBar.before(createMergeabilityRow({
+		action: <button type="button" className="btn rgh-update-pr-from-base-branch">Update branch</button>,
+		icon: <CheckIcon/>,
+		iconClass: 'completeness-indicator-success',
+		heading: 'This branch has no conflicts with the base branch',
+		meta: 'Merging can be performed automatically.',
```

Comment:
I added this meta line because without it it's a little misaligned, but I'm not totally sure it's a always _true_ statement exactly as GitHub defines it.